docker/README.md

```diff
@@ -31,6 +31,13 @@ MAX_CCU=1000
 MESSAGE_THREADS=0
 ```
 
+In order to take advantage of event loops / loop indication, the respective environment variables are available:
+
+``` 
+EVENT_LOOPS="true"
```

Comment:
Not "true", 8 for example (number of CPUs)
Can you remove the quotes from the values for consistency with the other variables in this doc.
Also please add the two variables in `docker/docker-compose.yml` as well.
You can add a reference to `https://github.com/meetecho/janus-gateway/pull/2450` for context.
Also please specify you will need to configure `adapter.setEventLoops(8)` (same value that what you configured in the environment variable) with naf-janus-adapter as well.

Reply:
Can you at least fix `EVENT_LOOPS="true"` by `EVENT_LOOPS=8` please? I'll merge and add some more text to the doc. Thanks.